spec/controllers/coronavirus_landing_page_controller_spec.rb

```diff
@@ -25,5 +25,54 @@
         expect(response.headers["Cache-Control"]).to eq("max-age=#{30.seconds}, public")
       end
     end
+
+    context "when testing national_applicability" do
+      render_views
+
+      before do
+        allow(Rails.env).to receive(:production?).and_return(true)
+      end
+
+      it "loads the production content item in production environments" do
+        expect(CoronavirusTimelineNationsContentItem).to_not receive(:load)
```

Comment:
Nitpick: this description doesn't seem to describe what is tested. It feels like it should be asserting on `ContentItem.find!("/coronavirus")` being called - I feel like there should be an inverse test too.

Reply:
I've added and extra test and re-done this one in a later commit.